src/covid_model_seiir_pipeline/pipeline/forecasting/specification.py

```diff
@@ -45,6 +45,7 @@ class ForecastData:
     regression_version: str = field(default='best')
     covariate_version: str = field(default='best')
     output_root: str = field(default='')
+    output_format: str = field(default='csv')
```

Comment:
What are you thoughts on emitting a warning or some other log message if this default is used?

Reply:
I think a deprecation warning is a good idea.